File: VB/CustomDrawCrosshairSample/Model/Order.vb
Imports System.ComponentModel.DataAnnotations
Imports System.ComponentModel.DataAnnotations.Schema

Namespace CustomDrawCrosshairSample.Model

    Public Partial Class Order

        Public Property OrderID As Integer

        <Column("EmployeeId")>
        Public Property EmployeeId As Integer?

        <Column(TypeName:="datetime")>
        Public Property OrderDate As Date

        Public Property Freight As Decimal?

        <ForeignKey("EmployeeId")>
        Public Overridable Property Employee As Employee
    End Class
End Namespace

File: CS/CustomDrawCrosshairSample/Model/Order.cs
namespace CustomDrawCrosshairSample.Model
{
    using System;
    using System.Collections.Generic;
    using System.ComponentModel.DataAnnotations;
    using System.ComponentModel.DataAnnotations.Schema;
    using System.Data.Entity.Spatial;

    public partial class Order
    {
        public int OrderID { get; set; }

        [Column("EmployeeId")]
        public int? EmployeeId { get; set; }

        [Column(TypeName = "datetime")]
        public DateTime OrderDate { get; set; }

        public decimal? Freight { get; set; }

        [ForeignKey("EmployeeId")]
        public virtual Employee Employee { get; set; }
    }
}
